LocationIQ Collection v2.0.0.postman_collection.json

```diff
@@ -1022,7 +1058,7 @@
 								{
 									"key": "countrycodes",
 									"value": "us",
-									"description": "Limit search to a specific of country.",
+									"description": "Limit search to a specific of country. Now supports multiple countrycodes. For example : \"us,ca\"",
```

Comment:
Limit search to a list of countries. Link to the notes section of docs for more info. Or specify examples inline.

Reply:
made changes